test/browser/pages/DVADetailsEntryPage.js

```diff
@@ -454,6 +454,12 @@ exports.DVADetailsEntryPage = class PlaywrightDevPage {
     );
   }
 
+  async viewSupportLink(supportFooterLink) {
+    await this.page.waitForLoadState("domcontentloaded");
```

Comment:
Could we combine this method and the below assertFooterLink method? That way we can test both the text and the link in the one step

Reply:
@ElliotMurphyGDS I have combined the 2 methods into one as requested.